portfolio/src/main/webapp/script.js

```diff
@@ -73,14 +73,22 @@ function createListElement(text) {
  * Fetches a list of comments from the server and adds it to the DOM.
  */
 async function getComments() {
-    const response = await fetch('/addcomment');
+    const response = await fetch(`/listcomment?num=${document.getElementById('num').value}`);
     const json = await response.json();
 
     const ulElement = document.getElementById('comments-list');
+    ulElement.textContent = '';
     for (comment of json) {
         ulElement.appendChild(createListElement(comment));
     }
     if(json.length === 0) {
         ulElement.textContent = 'No comments yet.';
     }
 }
+
+/**
+ * Sends a request to the servlet to delete all comments.
+ */
+function deleteComments() {
+    fetch('/deletecomment').then(res => getComments());
```

Comment:
Do we need to call `getComments()` after deleting them?

Reply:
It seems like we don't, thanks!